app/admin_dashboard/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask import Blueprint
+
+admin_dashboard = Blueprint('admin_dashboard', __name__)
```

Comment:
Potentially confusing if there is both an `admin` and a `admin_dashboard` blueprint. I created a separate blueprint for bulk upload (called `bulk`) since that seems like a fairly large feature with potentially a lot of routes and forms. Perhaps we should separate the rest of the admin features into a few different blueprints too?